packages/server/lib/socket-base.ts

```diff
@@ -490,6 +490,9 @@ export class SocketBase {
               return this.localBus.emit('cross:origin:finished', args[0])
             case 'cross:origin:automation:cookies:received':
               return this.localBus.emit('cross:origin:automation:cookies:received')
+            case 'request:sent:with:credentials':
+              // NOTE: this is currently a no-op until the server logic is implemented
+              return this.localBus.emit('request:sent:with:credentials', args[0])
```

Comment:
@AtofStryker Given @mjhenkes' changes will be landing soon to remove the localBut events for cross-origin, do you expect this will event will continue to emit as a a localBut event once that merges?

Reply:
yes everything should still emit as normal since the localBus is used heavily for automation tasks and we still have the setup in the server for the `cross:origin:automation:cookies` events